integrations/unity-csharp/Assets/VDK/UDSModel.cs

```diff
@@ -39,7 +38,31 @@ public string Path
         }
     }
 
-    // This gets called by VDKPPES if it isn't loaded already
+    private void Update()
+    {
+
+        if (geolocate)
+        {
+            this.transform.localPosition =  UDUtilities.UDtoGL *
+                new Vector3(
+                    (float)(header.baseOffset[0] + header.pivot[0] * header.scaledRange),
+                    (float)(header.baseOffset[1] + header.pivot[1] * header.scaledRange),
+                    (float)(header.baseOffset[2] + header.pivot[2] * header.scaledRange)
+                    );
+            transform.localPosition += geolocationOffset;
+            geolocate = false;
+        }
+    }
+    public Matrix4x4 getStoredMatrix() {
+        return new Matrix4x4(
+                new Vector4((float)header.storedMatrix[0], (float)header.storedMatrix[1], (float)header.storedMatrix[2], (float)header.storedMatrix[3]),
+                new Vector4((float)header.storedMatrix[4], (float)header.storedMatrix[5], (float)header.storedMatrix[6], (float)header.storedMatrix[7]),
+                new Vector4((float)header.storedMatrix[8], (float)header.storedMatrix[9], (float)header.storedMatrix[10], (float)header.storedMatrix[11]),
+                new Vector4((float)header.storedMatrix[12], (float)header.storedMatrix[13], (float)header.storedMatrix[14], (float)header.storedMatrix[15])
+                ) ;
```

Comment:
Indentation and splitting decisions are strange here.

Reply:
Indentation can be fixed (I assume you want the closing paren in line with the opening expressions rather than the contents and contents indented), unsure what to do regarding the splitting, I really don't want either of these expressions to be crammed on the one line as it makes the code extremely unreadable, declaring temporary variables doesn't improve it either IMO.